pom.xml

```diff
@@ -44,7 +44,7 @@
     <basepom.dependency-management.dependencies>true</basepom.dependency-management.dependencies>
     <basepom.dependency-management.plugins>true</basepom.dependency-management.plugins>
 
-    <basepom.test.timeout>120</basepom.test.timeout>
+    <basepom.test.timeout>900</basepom.test.timeout>
     <basepom.failsafe.timeout>300</basepom.failsafe.timeout>
```

Comment:
We may also want to bump this to 900 as well (since integration tests usually take as long as unit tests, if not longer)